README.md

```diff
@@ -215,7 +215,7 @@ To get the join command for kubeadm, run the following on your machine:
 
 ```powershell
 minikube ssh
-cd /var/lib/minikube/binaries/v1.23.3/
+cd /var/lib/minikube/binaries/v1.27.3/
```

Comment:
this version keeps changing, is there a way we can pick up what's installed on the control plane node automatically?

Reply:
That would be a good work around with the versioning. Let me explore this!